### PR TITLE
Added error message for untrusted certificate EWS V2

### PR DESCRIPTION
**Pull request by teizenman**
## Status
- [ ] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/29218

## Description
The status code is 503, the error is different than generic.

## Screenshots
image

### CODE REVIEW

File: Packs/EWS/Integrations/EWSv2/EWSv2.py [hunk 1]
```diff
@@ -2125,7 +2125,14 @@ def sub_main():
 
             error_message_simple += "You can try using 'domain\\username' as username for authentication. " \
                 if AUTH_METHOD_STR.lower() == 'ntlm' else ''
-        if "Status code: 503" in debug_log:
+
+        if "SSL: CERTIFICATE_VERIFY_FAILED" in debug_log:
+            # same status code (503) but different error.
+            error_message_simple = "Got timeout from the server. " \
```
Comment (glicht) [inline Packs/EWS/Integrations/EWSv2/EWSv2.py:2131]: Remove this line about timeout. As it is not timeout related. Let's use the following message:

`Certificate verification failed - This error may happen if the server certificate cannot be validated or as a result of a proxy that is doing SSL/TLS termination. It is possible to bypass certificate validation by checking 'Trust any certificate' in the instance settings.`

@kirbles19 please review this message.

**Review by glicht — CHANGES_REQUESTED** [review]

**Comment by teizenman** [comment]
@kirbles19 
Can you please review the error message here: https://github.com/demisto/content/blob/65ea732f7a4f90b10d913810397305c109faa6ba/Packs/EWS/Integrations/EWSv2/EWSv2.py#L2133:L2136

**Review by glicht — APPROVED** [review]

**Comment by glicht** [comment]
> @kirbles19
> Can you please review the error message here: https://github.com/demisto/content/blob/65ea732f7a4f90b10d913810397305c109faa6ba/Packs/EWS/Integrations/EWSv2/EWSv2.py#L2133:L2136

@kirbles19 waiting for you. Add the `docs-approved` tag when done.

**Comment by kirbles19** [comment]
@teizenman Looks good to me.

**Pull request merged**